cita-auth/build.rs

```diff
@@ -18,7 +18,9 @@ use std::env;
 
 use util::build_info::gen_build_info;
 
+const VERSION: &str = "1.0.0";
```

Comment:
What about put this at a public common type. then other modules use it directly?
```
pub const VERSION
```

Reply:
Yeah, that's a good option!
But I want to keep a separate version of the evolution capability for each service.

Reply:
> But I want to keep a separate version of the evolution capability for each service.

For this situation, just do not use the common version value and define own variable.
What i said can avoid that we miss some variables that shoud be changed to new version.

It's ok for me.  maybe discuss it later.

Reply:
Okay!
As you say, avoiding miss some variables be changed is a thinking aspect.
I have thought to define the `pub const VERSION` in cita-common, but I don't think this is a good option.